packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -156,7 +169,8 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
     // See https://docs.cryptowat.ch/rest-api/markets/ohlc for how this url is constructed.
     const ohlcUrl = [
       `https://api.cryptowat.ch/markets/${this.exchange}/${this.pair}/ohlc`,
-      `?after=${earliestHistoricalTimestamp}`,
+      `?before=${currentTime}`,
+      `&after=${earliestHistoricalTimestamp}`,
```

Comment:
I changed the restful query that gets sent to crypto watch to include the `before` field. Previously the query would return all data points up to the present. Now, we can define the upper time limit on prices returned.

For the operation of the normal bots, this will make no difference as the `before` is the `currentTime`.